.mdlrc

```diff
@@ -0,0 +1 @@
+rules "~MD036", "~MD013", "~MD024"
```

Comment:
@damacus 
Just curious on what basis we choose these rules only?

Reply:
These are rules turned off